packages/block-editor/src/components/block-mobile-toolbar/test/block-actions-menu.native.js

```diff
@@ -0,0 +1,311 @@
+/**
+ * External dependencies
+ */
+import {
+	fireEvent,
+	initializeEditor,
+	getBlock,
+	within,
+	getEditorHtml,
+	changeTextOfRichText,
+} from 'test/helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { getBlockTypes, unregisterBlockType } from '@wordpress/blocks';
+import { registerCoreBlocks } from '@wordpress/block-library';
+
+const INITIAL_HTML = `<!-- wp:paragraph -->
+<p></p>
+<!-- /wp:paragraph -->
+<!-- wp:spacer -->
+<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+<!-- wp:heading -->
+<h2 class="wp-block-heading"></h2>
+<!-- /wp:heading -->`;
```

Comment:
Does it make sense to insert a paragraph, spacer and heading manually as part of the setup, instead of relying on those blocks to always produce this markup? Seems like this could make the test more resilient against changes in the future.

Reply:
Sounds good, adding them each time won't increase too mcuh the running time of the tests so I'll update it 👍